scania-digital-design-system/sdds
# feat(popover-canvas): migrate popover-canvas component

**Describe pull-request**  
Migration popover-canvas

**Solving issue**  
Fixes: [AB#2412](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2412)

**How to test**  
1. Go to storybook link below
2. Check in components -> popover-canvas
3. Check that the design is aligned with the one in Figma.

**Screenshots**  
-

**Additional context**  
-
